app/commands.py

```diff
@@ -611,10 +616,31 @@ async def requests(ctx: Context) -> Optional[str]:
             l.append(f"Failed to find requested map ({map_id})?")
             continue
 
-        l.append(f"[{p.embed} @ {dt:%b %d %I:%M%p}] {bmap.embed}.")
+        l.append(f"#{id}: {bmap.embed}\n({dt:%b %d %I:%M%p}) Requested for "
+                 + f"{requested_status} by {p.embed}. "
+                 + f"{'No additional comment was specified.' if comment == '' else f'Additional comment: {comment}'}")
 
     return "\n".join(l)
 
+@command(Privileges.NOMINATOR, aliases=["dr"], hidden=True)
+async def declinerequest(ctx: Context) -> Optional[str]:
+    """Removes a map request by it's ID."""
+    if len(ctx.args) != 1:
+        return "Invalid syntax: !declinerequest <ID>"
```

Comment:
consistent with !mp map
```suggestion
        return "Invalid syntax: !declinerequest <beatmapid>"
```

Reply:
i wonder if this makes more sense to be based on /np though?

Reply:
From when I tried this on my server I felt like it was more handy to just put the ID there from the request so you wouldn't even have to download some maps, I feel like this is easier than nping

Reply:
maybe give the nominator the option to do both if they want?